src/blurredLocation.js

```diff
@@ -245,12 +261,35 @@ BlurredLocation = function BlurredLocation(options) {
       options.map.on('move', addScaleToListener);      
     }
     else {
-      $("#"+ options.scaleDisplay).text("");
+      $("#" + options.scaleDisplay).text("");
       options.map.off('move', addScaleToListener);
     }
   }
 
+  function getBlurryScale() {
+    var urban = options.blurryScaleDict["urban_"+getPrecision().toString()]
+    var rural = options.blurryScaleDict["rural_"+getPrecision().toString()]
+    return [urban, rural]
+  }
+
+  function addBlurryScale() {
+    blurryDescriptions = getBlurryScale();
+    $("#" + options.blurryScale).text("Urban description: "+ blurryDescriptions[0].toString() + " Rural description: " + blurryDescriptions[1].toString());
```

Comment:
Maybe the text could be:

```
This corresponds roughly to a _____ in an urban area, and _____ in a rural area.
```

I like this implementation!

Reply:
And just for ambiguity reasons, maybe `addBlurryScale` should be `displayBlurryScale`?

`blurryDescriptions` needs a `var`, i think?

Thanks, this looks super! Good implementation. Let's add some tests and then this should be good!

Reply:
Done!!